src/runtime/debug/outer-port-attachment.ts

```diff
@@ -54,40 +54,42 @@ export class OuterPortAttachment {
 
     // The slice discards the stack frame corresponding to the API channel
     // function, which is already being displayed in the log entry.
-    mapStackTrace(stackString, mapped => mapped.slice(1).map(frameString => {
-      // Each frame has the form '    at function (source:line:column)'.
-      // Extract the function name and source:line:column text, then set up
-      // a frame object with the following fields:
-      //   location: text to display as the source in devtools Arcs panel
-      //   target: URL to open in devtools Sources panel
-      //   targetClass: CSS class specifier to attach to the location text
-      let match = frameString.match(/^ {4}at (.*) \((.*)\)$/);
-      if (match === null) {
-        match = {1: '<unknown>', 2: frameString.replace(/^ *at */, '')};
-      }
+    if (mapStackTrace) {
```

Comment:
Could early-out instead, might be more readable given large block below.

Reply:
I took the approach of least-changes because handling of `mapStackTrace` is problematic at multiple levels and it's really something I need to discuss with @mykmartin. 

I went ahead and committed this as-is, because of perceived urgency getting working pipes-demo into ToT. Seems to me this can be smoothed over later.